node_modules/gh-users/tests/test-import.js

```diff
@@ -0,0 +1,302 @@
+/**
+ * Copyright (c) 2014 "Fronteer LTD"
+ * Grasshopper Event Engine
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+var _ = require('lodash');
+var assert = require('assert');
+var fs = require('fs');
+
+var AuthTestsUtil = require('gh-auth/tests/util');
+var ConfigTestsUtil = require('gh-config/tests/util');
+var RestAPI = require('gh-rest');
+var TestsUtil = require('gh-tests/lib/util');
+
+var UsersTestsUtil = require('./util');
+
+describe('Users - import', function() {
```

Comment:
We could just do 2 describes, one for `Users` and one for `Import`, rather than creating a text-based naming convention.